src/library/handleHttpResponseError.js

```diff
@@ -3,19 +3,13 @@ import language from '../language'
 import { getToastArguments } from './getToastArguments'
 
 const handleHttpResponseError = ({ error, callback, logoutMermaid }) => {
-  const errorStatus = error?.response?.status
-
-  if (!errorStatus) {
-    // If there is a general error which does not have a response status
-    // Don't throw an error. This should only ever encountered if axios is called after the the test is destroyed
-    toast.error(...getToastArguments(language.error.generic))
+  if (error) {
     console.error(error)
```

Comment:
we're just going to simplify here and console out any error if it is defined

Reply:
are we getting rid of a toast in this case? how will the user know if there is an http error?

Reply:
thats funther on down the file. Toasts will happen for a subset of http error types still and if there is a callback, that will be called to handle any other errors after